src/test/java/se/sundsvall/supportmanagement/service/mapper/ErrandMapperTest.java

```diff
@@ -272,6 +275,7 @@ void testToErrands() {
 				RESOLUTION,
 				DESCRIPTION,
 				ESCALATION_EMAIL,
+				BUSINESS_RELATED,
```

Comment:
you could throw a `hasNoNullFieldsOrProperties()`on this. It would have prevented the missing field in the first place 😃